test/unit/lib/much-rails/call_method_callbacks_tests.rb

```diff
@@ -0,0 +1,187 @@
+require "assert"
+
+require "much-rails/call_method_callbacks"
+
+module MuchRails::CallMethodCallbacks
+  class UnitTests < Assert::Context
+    desc "MuchRails::SaveService"
+    subject { unit_class }
+
+    let(:unit_class) { MuchRails::CallMethodCallbacks }
+
+    should "include MuchPlugin" do
+      assert_that(subject).includes(MuchPlugin)
+    end
+  end
+
+  class ReceiverTests < UnitTests
+    desc "receiver"
+    subject { receiver_class }
+
+    let(:receiver_class) {
+      Class.new do
+        include MuchRails::CallMethodCallbacks
+
+        attr_reader :values
+
+        def initialize
+          @values = []
+        end
+      end
+    }
+
+    let(:proc1) { -> {} }
+    let(:proc2) { -> {} }
+
+    should have_imeths :before_call, :prepend_before_call, :after_call
+    should have_imeths :prepend_after_call, :around_call, :prepend_around_call
+    should have_imeths :call
+
+    should "be configured as expected" do
+      assert_that(subject).includes(MuchRails::CallMethod)
+      assert_that(subject).includes(MuchRails::Config)
+    end
+
+    should "know its attributes" do
+      # before_callbacks
+      assert_that(
+        subject.much_rails_call_callbacks_config.before_callbacks.size)
+        .equals(0)
```

Comment:
Style thing: we are going to wrap closing `)` or `}` to the next line in general, so this becomes:

```suggestion
      assert_that(
        subject.much_rails_call_callbacks_config.before_callbacks.size
      ).equals(0)
```
The `.equals doesn't go on its own line b/c it isn't chained. Make sense? Update others please 🙏🏻 .